Dockerfile

```diff
@@ -15,6 +15,8 @@ RUN echo "*** Installing Compiler Explorer ***" \
         nodejs \
         make \
         git \
+        clang-11 \
```

Comment:
@arcticoder can you please remove the following? Actually the compilers are being installed in the parent image. I will try to add it, as also clang 12